apps/rush/src/actions/RushCommandLineParser.ts

```diff
@@ -90,6 +92,15 @@ export default class RushCommandLineParser extends CommandLineParser {
     }
   }
 
+  private _ensureEnvironment(): void {
+    /* tslint:disable-next-line:no-string-literal */
+    let environmentPath: string = process.env['PATH'];
+    environmentPath = path.join(this.rushConfiguration.commonTempFolder, 'node_modules', '.bin') +
```

Comment:
>this.rushConfiguration.commonTempFolder, 'node_modules', '.bin' [](start = 32, length = 63)

Is this always going to work?

What if two projects are using different versions of a tool? Neither will end up in the .bin folder, right?